src/app/Main/Body/InfoPage/Amrs/InfoPage-AmrsDialog.component.tsx

```diff
@@ -38,14 +38,15 @@ const starTextStyle = css`
 function createTableRowCells(row: AmrsTableData): JSX.Element[] {
     const tableCellList: JSX.Element[] = [];
     Object.keys(row).forEach((rowKey) => {
+        const amrRowKey = rowKey as AmrsTableDataKey

```

Comment:
So here it could be: rowKey as keyof AmrsTableData

Reply:
Done. Commit "Remove key type & define dialog outside map"